scripts/vm/hypervisor/xenserver/vmops

```diff
@@ -356,18 +356,25 @@ def allow_egress_traffic(session):
                 return 'false'
     return 'true'
 
+def getIpsetType():
+    try:
+        util.pread2(['/bin/bash', '-c', "ipset -v | grep 'protocol version: 6'"])
+        return 'nethash'
```

Comment:
For xenserver 6.5 the ipset version is 6.  For xenserver version less than 6.5 (i.e 6.2) the ipset version is 4.x. 
if ipset version is 6 then it uses nethash otherwise it uses iptreemap
The getIpsetType method is as below.
def getIpsetType():
    try:
        util.pread2(['/bin/bash', '-c', "ipset -v | grep 'protocol version: 6'"])
        return 'nethash'
    except:
        return 'iptreemap'